ci/e2e-tests.sh

```diff
@@ -592,6 +709,19 @@ ssh -i "$PWD/vm-ssh-key" "aziot@$vm_public_ip" '
     sudo usermod -aG aziotid aziot
 '
 
+if [ -f device-id.key.pem ] && [ -f device-id.pem ]; then
```

Comment:
Move this below the `device-id` one to be consistent with the order (symmkey code before x509 code).

Also, make it an `else if`.